web/controllers/rules/issue/time_estimation.ex

```diff
@@ -11,7 +11,7 @@ defmodule Dwylbot.Rules.TimeEstimation do
     in_progress = Enum.any?(labels, fn(l) -> l["name"] == "in-progress" end)
     estimation = labels
     |> Enum.map(fn(l) -> Regex.match?(~r/T\d{1,3}[mhd]/, l["name"]) end)
-    |> Enum.reduce(fn(x, acc) -> x || acc end)
+    |> Enum.reduce(false, fn(x, acc) -> x || acc end)
```

Comment:
I initialise the acc for the reduce function to ```false```. This avoid having an error if the list is empty.